src/app/components/AccountMenu/index.tsx

```diff
@@ -83,6 +83,7 @@ function AccountMenu({ title, subtitle, showOptions = true }: Props) {
                   selectAccount(accountId);
                 }}
                 disabled={loading}
+                selected={accountId === auth.account?.id}
```

Comment:
Would you mind extending this test with the new `selected` prop?:  
https://github.com/getAlby/lightning-browser-extension/blob/ed6bb1aa3ae39ed674abc8f686fb3f64ef867541/src/app/components/AccountMenu/index.test.tsx  

i.e. add a test "Current account is being highlighted"